src/Domain.LinnApps/Reports/ShortagesReportService.cs

```diff
@@ -127,6 +127,7 @@ public IEnumerable<ResultsModel> GetShortagesPlannerReport(int planner)
                     model.AddColumn("TotalBTReqt", "To achieve Ideal Build");
                     model.AddColumn("Supplier", "Supplier");
                     model.AddColumn("SupplierName", "Supplier Name");
+                    model.AddColumn("PlannerStory", "Planner Story");
```

Comment:
Don't call it planner story on the actual report. That's just what the table column happens to be called but they won't know what it is

Reply:
Noted! Will change it